lib/wpc_proto/internal_modules/proto_otap.h

```diff
@@ -0,0 +1,35 @@
+/* Wirepas Oy licensed under Apache License, Version 2.0
+ *
+ * See file LICENSE for full license details.
+ *
+ */
+#ifndef PROTO_OTAP_H_
+#define PROTO_OTAP_H_
+
+#include "generic_message.pb.h"
+#include "wpc.h"
```

Comment:
NIT:
Is wpc.h really needed here?
And generic_message.pb.h could be reduced to otap_message.pb.h

But I think I did the same in proto_config.h :-)

Reply:
Ok. (mais j'ai pas changé dans proto_config.h)